src/features/FoodAndBeverages/components/AddFoodAndBeverageModal/AddFoodAndBeverageModal.tsx

```diff
@@ -0,0 +1,89 @@
+    const [file, setFile] = useState<File | null>(null);
+
+    const dispatch = useDispatch();
+    return (
+        <Modal show={show} size="lg" aria-labelledby="contained-modal-title-vcenter" centered>
+            <Modal.Header closeButton>
+                <Modal.Title id="contained-modal-title-vcenter">Modal heading</Modal.Title>
+            </Modal.Header>
+            <Modal.Body>
+                <Form>
+                    <Form.Group as={Row} className="mb-3" controlId="formPlaintext5">
+                        <Form.Label column sm="2">
+                            Name
+                        </Form.Label>
+                        <Col sm="10">
+                            <Form.Control name="name" value={formValues.name} onChange={(e) => onChangeHandler(e)} />
+                        </Col>
+                    </Form.Group>
+
+                    <Form.Group as={Row} className="mb-3" controlId="formPlaintext5">
+                        <Form.Label column sm="2">
+                            Description
+                        </Form.Label>
+                        <Col sm="10">
+                            <Form.Control name="description" value={formValues.description} onChange={(e) => onChangeHandler(e)} />
+                        </Col>
+                    </Form.Group>
+
+                    <Form.Group as={Row} className="mb-3" controlId="formPlaintext5">
+                        <Form.Label column sm="2">
+                            Price
+                        </Form.Label>
+                        <Col sm="10">
+                            <Form.Control type="number" name="price" value={formValues.price} onChange={(e) => onChangeHandler(e)} />
+                        </Col>
+                    </Form.Group>
+
+                    <Form.Group as={Row} controlId="formFile" className="mb-3">
+                        <Form.Label column sm="2">
+                            Image
+                        </Form.Label>
+                        <Col sm="10">
+                            <Form.Control
+                                type="file"
+                                onChange={(e: React.ChangeEvent<HTMLInputElement>) => setFile(e.target.files ? e.target.files[0] : null)}
+                            />
+                        </Col>
+                    </Form.Group>
+                </Form>
+            </Modal.Body>
+            <Modal.Footer>
+                <Button type="button" onClick={() => dispatch(hideAddFoodAndBeverageModal())}>
+                    Close
+                </Button>
+                <Button
+                    type="button"
+                    onClick={async () => {
+                        if (file) {
+                            await createFoodAndBeverageHandler({ ...formValues, image: file });
+                            dispatch(hideAddFoodAndBeverageModal());
+                        }
+                    }}
```

Comment:
Same as above, move the handler to the variable if it has morew than 1 action inside.